Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -576,20 +714,51 @@ extension _FileManagerImpl {
                 .systemNumber : _writeFileAttributePrimitive(fsNumber, as: UInt.self)
             ]
         }
+#endif
     }
     
     func setAttributes(
         _ attributes: [FileAttributeKey : Any],
         ofItemAtPath path: String
     ) throws {
+        let mode = _readFileAttributePrimitive(attributes[.posixPermissions], as: UInt.self)
+        let immutable = _readFileAttributePrimitive(attributes[.immutable], as: Bool.self)
+        let appendOnly = _readFileAttributePrimitive(attributes[.appendOnly], as: Bool.self)
+
+#if os(Windows)
+        try path.withNTPathRepresentation {
+            if immutable != nil || appendOnly != nil {
+                // Setting these flags is not supported on this platform
+                throw CocoaError.errorWithFilePath(.featureUnsupported, path)
+            }
+
+            if let modification = attributes[.modificationDate] as? Date {
+                let seconds = modification.timeIntervalSince1970 + (Date.timeIntervalBetween1601AndReferenceDate - Date.timeIntervalBetween1970AndReferenceDate)
```

Comment:
Should we just have a `timeIntervalSince1601` property on `Date` for Windows that is `internal`?

Reply:
I like that!